From 8bb9d33d16f6004cc0420592b0dcf78571cdc746 Mon Sep 17 00:00:00 2001
From: Jan Palus <jpalus@fastmail.com>
Date: Mon, 17 Jun 2024 00:08:34 +0200
Subject: [PATCH] Fix memleak in cache after readlink

---
 cache.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cache.c b/cache.c
index 9436c5a7..70dc35b4 100644
--- a/cache.c
+++ b/cache.c
@@ -67,6 +67,7 @@ static void free_node(gpointer node_)
 {
 	struct node *node = (struct node *) node_;
 	g_strfreev(node->dir);
+	g_free(node->link);
 	g_free(node);
 }
 
