--- sshfs-fuse-1.5/Makefile.am.orig	2006-02-20 12:43:23.000000000 +0100
+++ sshfs-fuse-1.5/Makefile.am	2006-02-21 16:29:13.183053750 +0100
@@ -24,4 +24,4 @@
 	rm -f "$(DESTDIR)$(libdir)/sshnodelay.so"
 
 sshnodelay.so:
-	$(CC) -Wall -W -s --shared $(sshnodelay_libs) sshnodelay.c -o sshnodelay.so
+	$(CC) $(CFLAGS) -fPIC -shared $(sshnodelay_libs) sshnodelay.c -o sshnodelay.so
--- sshfs-fuse-1.5/Makefile.in.orig	2006-02-20 12:52:43.000000000 +0100
+++ sshfs-fuse-1.5/Makefile.in	2006-02-21 16:29:20.519512250 +0100
@@ -598,7 +598,7 @@
 	rm -f "$(DESTDIR)$(libdir)/sshnodelay.so"
 
 sshnodelay.so:
-	$(CC) -Wall -W -s --shared $(sshnodelay_libs) sshnodelay.c -o sshnodelay.so
+	$(CC) $(CFLAGS) -fPIC -shared $(sshnodelay_libs) sshnodelay.c -o sshnodelay.so
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
